pkg/activator/handler/handler.go

```diff
@@ -173,30 +113,26 @@ func (a *activationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	tryStart := time.Now()
-	err = a.throttler.Try(tryContext, revID, func() {
+	err := a.throttler.Try(tryContext, revID, func(dest string) error {
 		trySpan.End()
 		a.logger.Debugf("Waiting for throttler took %v time", time.Since(tryStart))
 
-		probeCtx, probeSpan := trace.StartSpan(r.Context(), "probe")
-		success, attempts := a.probeEndpoint(logger, r.WithContext(probeCtx), target)
-		probeSpan.End()
-
 		var httpStatus int
-		if success {
-			// Once we see a successful probe, send traffic.
-			attempts++
-			proxyCtx, proxySpan := trace.StartSpan(r.Context(), "proxy")
-			httpStatus = a.proxyRequest(w, r.WithContext(proxyCtx), target)
-			proxySpan.End()
-		} else {
-			httpStatus = http.StatusInternalServerError
-			w.WriteHeader(httpStatus)
+		target := url.URL{
+			Scheme: "http",
+			Host:   dest,
 		}
-
-		configurationName := revision.Labels[serving.ConfigurationLabelKey]
-		serviceName := revision.Labels[serving.ServiceLabelKey]
-		a.reporter.ReportRequestCount(namespace, serviceName, configurationName, name, httpStatus, attempts, 1.0)
+		// Once we see a successful probe, send traffic.
+		proxyCtx, proxySpan := trace.StartSpan(r.Context(), "proxy")
+		httpStatus = a.proxyRequest(w, r.WithContext(proxyCtx), &target)
+		proxySpan.End()
+
+		configurationName := "UNKNOWN"
+		serviceName := "UNKNOWN"
```

Comment:
Do we still need to fetch revision in order to get the config and service names?

Reply:
I put this back, but IMO we really don't want to be calling in to k8s clients at all during request path. We can figure out later how we want to remove it though.